### DISCUSSION TITLE
Smaller footer

### DISCUSSION THREAD

**Discussion by TheZoker** [facebook/docusaurus#4503] [Q&A] [answered]
Hi there,

Is there any option to have a smaller footer? Maybe the copyright text on the left and then some links on the right (just like the header)?

Something like:
![Screenshot_2021 03 23_18h38m00s_001_](https://user-images.githubusercontent.com/1368405/112192068-ed11ef00-8c06-11eb-8f46-df3ca96aa9c5.png)

Thanks!

**Comment by slorber** [comment, marked as answer]
You can use the swizzle cli feature and basically override `@theme/Footer` with your own implementation

**Reply by TheZoker** [reply]
Thanks! Is there any documentation available for this?

**Reply by armano2** [reply]
you can find it here https://docusaurus.io/docs/cli#docusaurus-swizzle-sitedir

```bash
yarn run swizzle @docusaurus/theme-classic Footer
```

**Comment by HonkingGoose** [comment]
@slorber Is swizzlling a thing that end-users should do now then?

Quote from https://docusaurus.io/docs/using-themes#swizzling-theme-components

> We would like to discourage swizzling of components until we've minimally reached a Beta stage. The components APIs have been changing rapidly and are likely to keep changing until we reach Beta. Stick with the default appearance for now if possible to save yourself some potential pain in future.

**Reply by slorber** [reply]
@HonkingGoose if you can do something without swizzling it's always better to avoid it, but if you want a custom footer ultimately there is no other choice.

The risk with swizzling is when we change the API of theme components, as it may break your code, and the whole theme becomes somehow a public API surface.

But if you implement fully a custom Footer component, not reading any input props, not using any other theme component inside it, you basically don't risk anything as it's unlikely we remove the theme/Footer component anyway

**Reply by slorber** [reply]
+ the cli reco with colors which comps are "safer" to swizzle, and Footer is one of them

![image](https://user-images.githubusercontent.com/749374/112303335-7293ae80-8c9c-11eb-834b-114ae28fd5ab.png)

**Reply by HonkingGoose** [reply]
Thanks for responding to me @slorber! 😄
